faq/index.md

```diff
@@ -10,31 +10,99 @@ type: markdown
+
+Detailed developer-level documentation is maintained for each component in github, alongside the code. Please visit each repository for those docs:
+
+*   [Envoy](https://lyft.github.io/envoy/docs/)
+*   [Manager](https://github.com/istio/manager/tree/master/doc)
+*   [Mixer](https://github.com/istio/mixer/tree/master/doc)
+
+#### Istio doesn't work - what do I do?
+
+Follow the [instructions](https://github.com/istio/istio/blob/master/CONTRIBUTING.md#issues) to open an issue [here](https://github.com/istio/istio/issues/new) or ask questions on [slack](https://istio-dev.slack.com/messages/C524NCGR1/). Our [users mailing list](https://groups.google.com/forum/#!forum/istio-users) is another great way to get help and answers. We also monitor [Stack Overflow](https://stackoverflow.com/questions/tagged/istio) for questions tagged with "istio".
+
+Additionally, we provide [Reference Guides](https://istio.io/docs/reference/) for all of the Istio components. These can be helpful when troubleshooting issues with configuration, etc.
+
+#### What does the Alpha release cover?
+
+The Alpha release of Istio provides an early preview of the intended functionality and user experience of our service mesh. We are hoping to solicit early feedback on direction and design decisions.
+
+The Alpha release includes the following features:
+
+*   Simple command-line installation into a kubernetes cluster
+*   Scripted proxy injection with traffic capture via iptables
+*   L7 traffic routing rules
+*   In-cluster load balancing for HTTP, GRPC & TCP
+*   Cluster Ingress and Egress
+*   Fault injection
+*   In-memory rate limiting
+*   L7 Metrics and Logs collection
+*   Secure service-to-service authentication with strong identity
+*   Pluggable policy layer and configuration API
+
+#### Does Istio Auth support authorization?
+
+Not currently - but we are working on it. At the moment, we only support the kubernetes service account as the principal identity in Istio Auth. We are investigating using [JWT](https://jwt.io/) together with mutual TLS to support enhanced authentication and authorization.
+
+#### Does Istio Auth use kubernetes secrets?
+
+Yes. The key and certificate distribution in Istio Auth is based on [kubernetes secrets](https://kubernetes.io/docs/concepts/configuration/secret/).
+
+Secrets have known [security risks](https://kubernetes.io/docs/concepts/configuration/secret/#risks). The kubernetes team is working on [several features](https://docs.google.com/document/d/1T2y-9geg9EfHHtCDYTXptCa-F4kQ0RyiH-c_M1SyD0s) to improve kubernetes secret security, from secret encryption to node-level access control. And as of version 1.6, kubernetes introduces [RBAC authorization](https://kubernetes.io/docs/admin/authorization/rbac/), which can provide fine-grained secrets management.
+
+#### What kind of traffic does Istio Auth support?
+
+For Alpha, we only support http traffic. And we are actively working on supporting more kinds of traffic like SQL, etc.
```

Comment:
Made it HTTP. Assuming HTTP, as input was direct from auth team. They would have to update if it should be HTTP(S).